src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/hmcts/CourtCaseEventsProcessor.kt

```diff
@@ -100,7 +100,7 @@ class CourtCaseEventsProcessor(
         EventKeys.RECORD_COUNT to pageablePersonEntities.totalElements.toString(),
         EventKeys.EVENT_TYPE to LIBRA_COURT_CASE.name,
         EventKeys.MESSAGE_ID to sqsMessage.messageId,
-        EventKeys.SEARCH_VERSION to "1",
+        EventKeys.SEARCH_VERSION to "1.1",
```

Comment:
why not 2?

Reply:
As it probably what v1 should have been

Reply:
I just think 1,2,3 is simpler than using semver here, although as long as we can tie it back to the algorithm it doesn't matter as long as it changes each time